src/components/gameHeaderComponent/gameHeader.spec.js

```diff
@@ -11,5 +11,5 @@ describe('Header Component', () => {
     const gameHeaderDiv = document.createElement('div');
     ReactDOM.render(<GameHeader />, gameHeaderDiv);
     ReactDOM.unmountComponentAtNode(gameHeaderDiv);
-  });
+  });  
```

Comment:
I would remove trailing whitespace. I would also look into adding a linting rule to enforce no trailing whitespace.

Reply:
Working on the custom eslint rule now

Reply:
Still a little stuck on this one `"no-trailing-spaces": ["error", {"skipBlankLines": false}]`. This is what I currently have, but doesn't seem to working the way I expect.

Reply:
Never mind, I understand now, it's working